const.gen.go

```diff
@@ -147,357 +73,445 @@ const (
+	// DomainBandwidthInPeak as defined in libvirt/libvirt-domain.h:1691
 	DomainBandwidthInPeak = "inbound.peak"
-	// DomainBandwidthInBurst as defined in libvirt/libvirt-domain.h:1705
+	// DomainBandwidthInBurst as defined in libvirt/libvirt-domain.h:1698
 	DomainBandwidthInBurst = "inbound.burst"
-	// DomainBandwidthInFloor as defined in libvirt/libvirt-domain.h:1712
+	// DomainBandwidthInFloor as defined in libvirt/libvirt-domain.h:1705
 	DomainBandwidthInFloor = "inbound.floor"
-	// DomainBandwidthOutAverage as defined in libvirt/libvirt-domain.h:1719
+	// DomainBandwidthOutAverage as defined in libvirt/libvirt-domain.h:1712
 	DomainBandwidthOutAverage = "outbound.average"
-	// DomainBandwidthOutPeak as defined in libvirt/libvirt-domain.h:1726
+	// DomainBandwidthOutPeak as defined in libvirt/libvirt-domain.h:1719
 	DomainBandwidthOutPeak = "outbound.peak"
-	// DomainBandwidthOutBurst as defined in libvirt/libvirt-domain.h:1733
+	// DomainBandwidthOutBurst as defined in libvirt/libvirt-domain.h:1726
 	DomainBandwidthOutBurst = "outbound.burst"
-	// DomainIothreadPollMaxNs as defined in libvirt/libvirt-domain.h:2038
+	// DomainIothreadPollMaxNs as defined in libvirt/libvirt-domain.h:2031
 	DomainIothreadPollMaxNs = "poll_max_ns"
-	// DomainIothreadPollGrow as defined in libvirt/libvirt-domain.h:2048
+	// DomainIothreadPollGrow as defined in libvirt/libvirt-domain.h:2041
 	DomainIothreadPollGrow = "poll_grow"
-	// DomainIothreadPollShrink as defined in libvirt/libvirt-domain.h:2059
+	// DomainIothreadPollShrink as defined in libvirt/libvirt-domain.h:2052
 	DomainIothreadPollShrink = "poll_shrink"
-	// PerfParamCmt as defined in libvirt/libvirt-domain.h:2251
+	// PerfParamCmt as defined in libvirt/libvirt-domain.h:2245
 	PerfParamCmt = "cmt"
-	// PerfParamMbmt as defined in libvirt/libvirt-domain.h:2262
+	// PerfParamMbmt as defined in libvirt/libvirt-domain.h:2256
 	PerfParamMbmt = "mbmt"
-	// PerfParamMbml as defined in libvirt/libvirt-domain.h:2272
+	// PerfParamMbml as defined in libvirt/libvirt-domain.h:2266
 	PerfParamMbml = "mbml"
-	// PerfParamCacheMisses as defined in libvirt/libvirt-domain.h:2282
+	// PerfParamCacheMisses as defined in libvirt/libvirt-domain.h:2276
 	PerfParamCacheMisses = "cache_misses"
-	// PerfParamCacheReferences as defined in libvirt/libvirt-domain.h:2292
+	// PerfParamCacheReferences as defined in libvirt/libvirt-domain.h:2286
 	PerfParamCacheReferences = "cache_references"
-	// PerfParamInstructions as defined in libvirt/libvirt-domain.h:2302
+	// PerfParamInstructions as defined in libvirt/libvirt-domain.h:2296
 	PerfParamInstructions = "instructions"
-	// PerfParamCPUCycles as defined in libvirt/libvirt-domain.h:2312
+	// PerfParamCPUCycles as defined in libvirt/libvirt-domain.h:2306
 	PerfParamCPUCycles = "cpu_cycles"
-	// PerfParamBranchInstructions as defined in libvirt/libvirt-domain.h:2322
+	// PerfParamBranchInstructions as defined in libvirt/libvirt-domain.h:2316
```

Comment:
We might have no control over this, but I've mixed feelings about the line numbers. On the one hand, could be useful, and they are auto-generated, on the other hand, it makes it really hard to review the generated code to see if its purely additive. I wonder if it would be possible to have the comment have only stable info, file path, and perhaps the name of the symbol the value is pulled from, if the "C" symbol is not exactly `PerfParamBranchInstructions` ?

Reply:
I like this suggestion. I could look into hacking `c-for-go` to make this configurable.